src/order_transfer/presentation/websocket.go

```diff
@@ -0,0 +1,17 @@
+package presentation

```

Comment:
esto deberia ir dentro de infra, aunque sea en una carpeta presentation

Reply:
ok

Reply:
the same happens on [DEV-42](https://github.com/HyperloopUPV-H8/Backend-H8/pull/46) where we have the same structure, should I change it there too?